appsec/src/extension/commands_helpers.c

```diff
@@ -648,6 +678,103 @@ bool dd_command_process_metrics(mpack_node_t root, zend_object *nonnull span)
     return true;
 }
 
+static void _handle_telemetry_metric(const char *nonnull key_str,
```

Comment:
Not sure if it is a good practice or I made it up but, should this early declaration be at the top?

Reply:
You put these at the top if you use these multiple times at various places. For a single use like here, there's no reason to.